pyflagser/flagser.py

```diff
@@ -41,6 +41,17 @@ def flagser(flag_matrix, min_dimension=0, max_dimension=np.inf, directed=True,
         often ``100,000``. Increase for higher precision, decrease for faster
         computation. A negative value computes highest possible precision.
 
+    filtration : string, optional, default: ``max``
+        filtration algorithm use the specified algorithm to compute the
```

Comment:
@MonkeyBreaker could you fix the wording of the first sentence?

Reply:
Is `Use the specified algorithm to compute the filtration.` better ?

Reply:
I was thinking just "Algorithm determining the filtration."